crypto/init.c

```diff
@@ -487,6 +484,9 @@ int OPENSSL_init_crypto(uint64_t opts, const OPENSSL_INIT_SETTINGS *settings)
         aloaddone = 1;
     }
 
+    if (!ossl_init_casecmp())
```

Comment:
Is this necessary?
It's going to be initialised if called, so there doesn't seem much point also initialising it here.

Removing it here would also allow some clean up (reduction in the number of functions) in the initialisation code.

Reply:
I always prefer proper initialization when OPENSSL_init_crypto() is called to lazy initialization. So I'd like to keep it here.